ios/RCTPSPDFKit/RCTPSPDFKitManager.m

```diff
@@ -31,6 +31,7 @@ @implementation RCTPSPDFKitManager
 
   UINavigationController *navigationController = [[UINavigationController alloc] initWithRootViewController:pdfViewController];
 
+  navigationController.modalPresentationStyle = UIModalPresentationOverFullScreen;
```

Comment:
I agree with Douglas that `UIModalPresentationFullScreen` might be a better fit here.

```suggestion
  navigationController.modalPresentationStyle = UIModalPresentationFullScreen;
```